src/data_warehouse/cli/monitor.py

```
@@ -3,146 +1,219 @@
 import click
-
+import sqlite3
+import pandas as pd
+from tabulate import tabulate
+from pathlib import Path
```

Ruff (0.8.2)

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)